src/panels/lovelace/common/create-entity-row-element.js

```diff
@@ -0,0 +1,74 @@
+import fireEvent from '../../../common/dom/fire_event.js';
+
+import stateCardType from '../../../common/entity/state_card_type.js';
+import computeDomain from '../../../common/entity/compute_domain.js';
+import { DOMAINS_HIDE_MORE_INFO } from '../../../common/const.js';
+
+import createErrorCardConfig from './create-error-card-config.js';
+
+const CUSTOM_TYPE_PREFIX = 'custom:';
+
+function _createElement(tag, config, stateObj, hass) {
+  const element = document.createElement(tag);
+  try {
+    // eslint-disable-next-line
+    'setConfig' in element && element.setConfig(config);
+  } catch (err) {
+    // eslint-disable-next-line
+    console.error(tag, err);
+    // eslint-disable-next-line
```

Comment:
What is disabled here ?

I feel like you are too eager with disabling things instead of fixing things.

Reply:
This is part of the original code. Didn't touch this I think

Reply:
https://github.com/home-assistant/home-assistant-polymer/blob/master/src/panels/lovelace/common/create-card-element.js

I just took over the concept and disabled a few more things then as it was done in the original code